index.js

```diff
@@ -83,9 +87,15 @@
         }, $this.data('load-after'));
       });
     } else {
-      whenScroll(['within 300px of', $newElement[0]], function () {
-        loadNow($this, $newElement);
-      }, true);
+      // This is essential for firing the event to a DOM immediately, as set by
```

Comment:
I <3 the mysterious third parameter

Should this be fixed in when-scroll, not whtevr?